src/index.js

```diff
@@ -1,14 +1,70 @@
 // import _ from 'lodash';
 import './style.css';
 
-// function component() {
-//   const element = document.createElement('div');
+// const gameID = 'AsHkmrIRRRVDeNZNfO3f';
+const playername = document.getElementById('playername');
+const playerscore = document.getElementById('playerscore');
+const scoresubmit = document.getElementById('scoresubmit');
 
-//   // Lodash, now imported by this script
-//   element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-//   element.classList.add('hello');
+async function getScores() {
+  const response = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/{$gameID}/scores/');
+  if (response.ok) {
+    const data = await response.json();
+    const ul = document.getElementById('list');
+    ul.innerHTML = '';
+    data.result.forEach((score) => {
+      const li = document.createElement('li');
+      li.innerHTML = `${score.user}: ${score.score}`;
+      ul.appendChild(li);
+    });
+  } else {
+    // console.log('Error');
+  }
+}
 
-//   return element;
+async function postScores(name, score) {
+  const response = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/{$gameID}/scores/', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({ user: name, score }),
+  });
+  if (response.ok) {
+    getScores();
+  } else {
+    // console.log('Error'); for error handling.
+  }
+  const data = await response.json();
+  return data;
+}
+
+scoresubmit.addEventListener('click', () => {
+  const name = playername.value;
+  const score = parseInt(playerscore.value, 10);
+  postScores(name, score);
+  playername.value = '';
+  playerscore.value = '';
+});
+
+// function createScoreList() {
+
+//   const scoreList = document.getElementById('ul');
+
+//   scoreList.setAttribute('id', 'score-list');
+//   return scoreList;
+// }
+
+// function createScoreListItem(name, score) {
+//   const scoreListItem = document.createElement('li');
+//   scoreListItem.innerHTML = `${name}: ${score}`;
+//   return scoreListItem;
 // }
 
-// document.body.appendChild(component());
\ No newline at end of file
+// function createScoreListItems(scores) {
+//   const scoreListItems = [];
+//   scores.forEach((score) => {
+//     scoreListItems.push(createScoreListItem(score.user, score.score));
+//   });
+//   return scoreListItems;
+// }
```

Comment:
- [ ] Kindly consider removing any commented or unused code from your `codebase` as it can make your code messy and difficult to read.